### PR TITLE
Changed naming of contracts to be more telling

### PR DESCRIPTION
**Pull request by erduna**
## Description

As agreed with team Joules and @Sondergaard we have renamed our contract names to better reflect future events.

## References

* #0000

### CODE REVIEW

**Review by Korgath — APPROVED** [review]

**Review by kft — CHANGES_REQUESTED** [review]
Public interfaces should be alligned internally in team prior to be exposed. Why are the events needed ? And do we expose internal properties ? (MeteingPointId, TransactionId)

**Comment by erduna** [comment]
> Public interfaces should be alligned internally in team prior to be exposed. Why are the events needed ? And do we expose internal properties ? (MeteingPointId, TransactionId)

I have aligned it with other teams, with the architect and with the business. We can have another internal talk if needed :)

**Review by Korgath — APPROVED** [review]

File: docs/contracts.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+### EnergySupplierChanged
+
+This message is sent out when an energy supplier has been changed.
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| MeteringPointId | [string](#string) |  | Unique metering point identification |
+| EnergySupplierGLN | [string](#string) |  | Unique Energy Supplier identification. |
+| EffectiveDate | [string](#string) |  | Date which the change of supplier goes into effect. |
```
Comment (MrDach) [inline docs/contracts.md:28]: Nit: Date **on** which

File: docs/contracts.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+### FutureEnergySupplierChangeAccepted
+
+This message is sent out when a future energy supplier change is accepted. It informs who will be the future energy supplier of the respective metering point
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| MeteringPointId | [string](#string) |  | Unique metering point identification |
+| EnergySupplierGLN | [string](#string) |  | Unique Energy Supplier identification |
+| EffectiveDate | [string](#string) |  | Date in which the supplier change will go into effect |
```
Comment (MrDach) [inline docs/contracts.md:45]: Nit: Date **on** which

**Review by MrDach — APPROVED** [review]
Only two minor suggestions but looks good otherwise.

**Review by MrDach — APPROVED** [review]

**Review by Korgath — APPROVED** [review]

**Pull request merged**